### PR TITLE
gcscopy e2e tests

### PR DESCRIPTION
**Pull request by priyabhatnagar25**

### CODE REVIEW

File: src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSMove.java
Comment (rahuldash171) [inline src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSMove.java:1]: Why this file is moved to gcs/stepdesign ? It was previously present inside gcsmove/stepdesign .

File: src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSDoneFileMarker.java
Comment (rahuldash171) [inline src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSDoneFileMarker.java:1]: GCSDone file marker feature was added for gcsMove , why it is present inside gcs/stepdesign ?

File: src/e2e-test/resources/errorMessage.properties
Comment (rahuldash171) [inline src/e2e-test/resources/errorMessage.properties:1]: Error messages are incomplete sentences . What is the reason ?

File: src/e2e-test/java/io/cdap/plugin/gcsmove/stepsdesign/package-info.java
Comment (rahuldash171) [inline src/e2e-test/java/io/cdap/plugin/gcsmove/stepsdesign/package-info.java:1]: Why this file is deleted from here (gcsMove/stepdesign) and added inside gcs/stepdesign ?

File: src/e2e-test/java/io/cdap/plugin/gcscopy/Locators/GCSCopyLocators.java
Comment (rahuldash171) [inline src/e2e-test/java/io/cdap/plugin/gcscopy/Locators/GCSCopyLocators.java:1]: These are generic locators which are used in gcsMove also . Do we have to create them again ? Can't we use the generic ones ?

File: src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/package-info.java
Comment (rahuldash171) [inline src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/package-info.java:1]: We should not change any old file , this file is modified . Why step design for GCS was removed and move is added here . package-info.java file for each plugin should be present in the respective folder .

**Review by rahuldash171 — COMMENTED** [review]
Review comments added

File: src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSMove.java
Comment (priyabhatnagar25) [reply src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSMove.java:1]: reverted

File: src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSDoneFileMarker.java
Comment (priyabhatnagar25) [reply src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSDoneFileMarker.java:1]: reverted

File: src/e2e-test/resources/errorMessage.properties
Comment (priyabhatnagar25) [reply src/e2e-test/resources/errorMessage.properties:1]: For Some cases it doesnt take full statement thats why I not taken full error messages.

File: src/e2e-test/java/io/cdap/plugin/gcsmove/stepsdesign/package-info.java
Comment (priyabhatnagar25) [reply src/e2e-test/java/io/cdap/plugin/gcsmove/stepsdesign/package-info.java:1]: reverted

File: src/e2e-test/java/io/cdap/plugin/gcscopy/Locators/GCSCopyLocators.java
Comment (priyabhatnagar25) [reply src/e2e-test/java/io/cdap/plugin/gcscopy/Locators/GCSCopyLocators.java:1]: So there in GCS Move its written that these are the locators for GCS Move , so I just changed their names to GCS copy and used them in the scenarios

File: src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/package-info.java
Comment (priyabhatnagar25) [reply src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/package-info.java:1]: revertred

File: src/e2e-test/java/io/cdap/plugin/gcscopy/Locators/GCSCopyLocators.java
Comment (priyabhatnagar25) [reply src/e2e-test/java/io/cdap/plugin/gcscopy/Locators/GCSCopyLocators.java:1]: As Discussed we can actions, locators and step definition for gcscopy separate

**Pull request merged**